ambry-frontend/src/main/java/com.github.ambry.frontend/FrontendMetrics.java

```diff
@@ -137,6 +137,16 @@
   public final Histogram blobStorageServiceStartupTimeInMs;
   public final Histogram blobStorageServiceShutdownTimeInMs;
 
+  // Account
+  public final Counter oldPutUnrecognizedAccountCount;
```

Comment:
lets not have metrics with names like "old" "new" - names are too relative. Let's chat about this so that we have metrics that are useful and long term.